core/src/components/datetime/datetime.tsx

```diff
@@ -171,6 +174,24 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() disabled = false;
 
+  /**
+   * Formatting options, separated by date and time.
+   */
+  @Prop() formatOptions?: DatetimeFormatOptions;
+
+  @Watch('formatOptions')
+  protected formatOptionsChanged() {
+    this.errorIfTimeZoneProvided();
+  }
+
+  get dateFormatOptions(): Intl.DateTimeFormatOptions | undefined {
```

Comment:
Can we add a comment block for these helper getters that explains why they exist/what they do? 

I think we might want to track a technical debt task here as well to evaluate if we can drop these in the future. There is some additional configuration/consideration that is causing this to operate differently than the typescript playground that we demonstrated in the design document.